modules/virt-deleting-virt-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * virt/install/uninstalling-virt.adoc
+
+[id="virt-deleting-virt-cli_{context}"]
+= Deleting the {VirtProductName} using the CLI
+
+You can delete {VirtProductName} by using the CLI.
+
+.Prerequisites
+
+- `oc` command installed on workstation.
+
+- Access to an {product-title} cluster using an account with
```

Comment:
This makes it sound like you need access to any cluster, not the one that you have Virt installed on.

Reply:
Updated this content.